### ISSUE TITLE
Feature Request: Output bmfont format

### ISSUE THREAD

**Issue by htdag** [Chlumsky/msdf-atlas-gen#5]
Hi,

Export fnt (txt) or json with bmfont compatible format.

Is it doable?

Thanks in advance.

**Comment by Chlumsky** [comment]
Yes, it is doable, I actually wanted to do it but didn't have time.

**Comment by miltoncandelero** [comment]
Similar to this idea I was trying to parse the outputted json directly however I found that `glyph.advance` is in some kind of arcane ratio and not in pixels.

@Chlumsky are there any docs on how to make sense of the numbers exported in the glyph of the json export?
![image](https://user-images.githubusercontent.com/15253010/132423409-653c41e1-4490-4b21-b2cc-79e9810f76a9.png)

**Comment by Chlumsky** [comment]
Multiply it by the `"size"` entry and you got it in pixels. Very arcane indeed.

**Comment by miltoncandelero** [comment]
That makes sense, sorry that I came up a bit rough I had fought that for hours and I was frustrated.

You made a great tool! ✨

**Comment by pha3z** [comment]
@miltoncandelero 

Did you complete a parser that converts the json format into bmfont format?

I would also like to work in bmfont.

**Comment by dtaylorus** [comment]
I created a simple bmfont converter that works for my needs at https://github.com/hotchaipro/msdf-atlas-bmfont
